### DISCUSSION TITLE
Suggestions for add popup menu

### DISCUSSION THREAD

**Discussion by vasigias** [libretro/docs#791] [Ideas]
Suggestions for add popup menu with save/load or exit when leave from a game 
![retroarch](https://user-images.githubusercontent.com/6636759/209981286-1ff3cd19-e435-4855-934a-082a180cf275.png)

**Comment by hizzlekizzle** [comment]
This is exactly the purpose of the quick menu, it just has grown over time to include a lot more stuff. Those important options are right near the top, though, where they belong.

**Reply by vasigias** [reply]
happy New Year
I suggest that by simply clicking to exit the game, for example superpang, to display this menu with the options so that you don't need to have more buttons or search to find these options, which I consider quite useful
